e2e/tests/ui-driven/src/helpers/addComponent.ts

```diff
@@ -55,11 +55,11 @@ const createBaseComponent = async (
       break;
     case ComponentType.AddressInput:
       await page.getByPlaceholder("Title").fill(title || "");
-      await page.getByPlaceholder("Data Field").fill(options?.[0] || "");
+      await page.locator("id=data-field-autocomplete").fill(options?.[0] || "");
```

Comment:
E2E still are ultimately failing because nodes with a `required` data field are not being correctly populated -I've tried a couple iterations here, but still not landed on the right solution. 

Opening for wider review in the meantime and any pointers very welcome! 

Here's what I've tried so far:
- The placeholder text has changed from `"Data Field"` to `"Data field"`, but `getByPlaceholder()` will now match more than one element (MUI Autocompletes have many nested elements)
![Screenshot from 2024-12-18 21-46-20](https://github.com/user-attachments/assets/d5c44525-2325-4924-bb3d-ff6e5d8e68fd)
- So, `"getByRole("combobox", { name: "Data field" })` _should_ exclusively be matching the `input` element
- On the first component, there will be no flow schema or suggestions yet and the `popupIcon` arrow is disabled, so there's no dropdown of options to `.click()` or open/expand - but I'm guessing I'm still using the wrong combo of `.fill`/`.click` here to mimic typing input ??